src/main/java/com/launchacademy/reviews/controllers/TrailsApiV1Controller.java

```diff
@@ -81,4 +82,11 @@ public ResponseEntity updateTrail(@RequestBody @Valid Trail trail, BindingResult
       return new ResponseEntity(HttpStatus.ACCEPTED);
     }
   }
+
+  @DeleteMapping("/{id}/delete")
+  public void deleteTrail(@PathVariable Integer id) {
+    if(trailService.findById(id).isPresent()){
```

Comment:
While this work, the is isPresent is better served in the service.  Also, it would be useful to return some context as to whether the delete successfully completed or not.  Not a deal breaker, but you or the user won't have any idea as to what happened unless a UI change is obvious